rental_object.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
```

Comment:
- [ ] [REQUIRED] Good job so far! **Getters** and **Setters** for `@book` and `@person` instance variables should be present like below and please remove line `2` and change `attr_accessor` values:

```ruby
attr_accessor :date, :book, :person
```